eigrpd/eigrp_packet.c

```diff
@@ -172,7 +172,7 @@ int eigrp_check_md5_digest(struct stream *s,
 	uint8_t *ibuf;
 	size_t backup_end;
 	struct TLV_MD5_Authentication_Type *auth_TLV;
-	struct eigrp_header *eigrph;
+	//struct eigrp_header *eigrph;
```

Comment:
Please delete these lines rather than comment them out. Same comment for the one further down.